apps/condo/cypress/plugins/index.js

```diff
@@ -8,10 +8,11 @@ const {
     createTestForgotPasswordAction,
     ConfirmPhoneAction,
     makeLoggedInClient,
+    makeClientWithSupportUser,
 } = require('@condo/domains/user/utils/testSchema')
 const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
 const { OrganizationEmployee } = require('@condo/domains/organization/utils/testSchema')
-const { createTestTicket } = require('@condo/domains/ticket/utils/testSchema')
+const { createTestTicket, createTestTicketClassifierRule } = require('@condo/domains/ticket/utils/testSchema')
```

Comment:
You can use existing classifier rules as they are created in migration

Reply:
Yup, but I try to use fully generated data inside this tests